### PR TITLE
Add IPNS support and endpoint

### PR DESCRIPTION
**Pull request by Man-Jain**

### CODE REVIEW

File: packages/storage/src/spheron-api/interfaces.ts [hunk 1]
```diff
@@ -159,4 +159,14 @@ export interface UsageWithLimitsWithSkynet extends UsageWithLimits {
   storageSkynetLimit?: number; // Bytes
 }
 
+export interface IPNSPublishResponse {
+  _id: string;
```
Comment (shomix) [inline packages/storage/src/spheron-api/interfaces.ts:163]: just need to create a new interface where this _id is called `id` and add mappings to it.

**Review by shomix — COMMENTED** [review]
Also need to add this methods to the README, increase the version in package json and readme.

File: packages/storage/src/spheron-api/interfaces.ts (hunk 1 above)
Comment (shomix) [reply packages/storage/src/spheron-api/interfaces.ts:163]: also `publishedDeploymentId` should be mapped to `publishedUploadId`

File: packages/storage/src/spheron-api/index.ts [hunk 2]
```diff
@@ -190,6 +192,84 @@ class SpheronApi {
     return usage;
   }
 
+  public async publishIPNS(deploymentId: string): Promise<IIPNSName> {
+    try {
+      const resp = await this.sendApiRequest<IPNSPublishResponse>(
```
[the remark this answers is not in the document]
Comment (shomix) [reply packages/storage/src/spheron-api/index.ts:197]: same for upload

**Review by shomix — APPROVED** [review]

**Review by dusanstanis — APPROVED** [review]

**Pull request merged**